packages/utils/parcelforvscode/README.md

```diff
@@ -6,3 +6,39 @@ This extension shows errors, warnings and other diagnostics inline in VS Code.
 
 1. Install this extension
 2. In your project, install `@parcel/reporter-lsp` and run Parcel with that reporter, e.g. `parcel src/index.html --reporter @parcel/reporter-lsp`.
+
+## Development Debugging
```

Comment:
This is the user-facing readme that will be visible in the extension marketplace, e.g. https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer

So this should probably be in a `CONTRIBUTING:md` or something like that

Reply:
Done